### PR TITLE
Central Repo NPE

### PR DESCRIPTION
**Pull request by dannysmyda**

### CODE REVIEW

**Comment by dannysmyda** [comment]
Looking at SQLiteCentralRepo, all of `connect()` is synchronized. This leads me to believe it was just a simple mistake.

File: Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/PostgresCentralRepo.java [hunk 1]
```diff
@@ -198,11 +198,11 @@ protected Connection connect() throws CentralRepoException {
             if (connectionPool == null) {
                 setupConnectionPool();
             }
-        }
```
Comment (dannysmyda) [inline Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/PostgresCentralRepo.java:201]: This `}` ends the synchronized block. I've moved it down to line 205.

File: Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/PostgresCentralRepo.java [hunk 2]
```diff
@@ -198,11 +198,11 @@ protected Connection connect() throws CentralRepoException {
...
-        try {
-            return connectionPool.getConnection();
-        } catch (SQLException ex) {
-            throw new CentralRepoException("Error getting connection from connection pool.", Bundle.PostgresEamDb_connectionFailed_message(), ex); // NON-NLS
+            try {
+                return connectionPool.getConnection();
+            } catch (SQLException ex) {
+                throw new CentralRepoException("Error getting connection from connection pool.", Bundle.PostgresEamDb_connectionFailed_message(), ex); // NON-NLS
+            }
```
Comment (rcordovano) [inline Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/PostgresCentralRepo.java:205]: I'd be really, really surprised if the connection pool getConnection() method is not thread-safe.

Comment (dannysmyda) [reply Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/PostgresCentralRepo.java:205]: Not getConnection(), but the interaction with the `connectionPool` variable is not thread-safe

Comment (rcordovano) [reply Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/PostgresCentralRepo.java:205]: Yes, of course. You are correct.

**Pull request merged**